### PR TITLE
Add labelauty@1.1.0 w/ git auto-update

### PR DESCRIPTION
**Pull request by menuka94**
Pull request for issue: #9926

# Profile of the lib
 * Git repository: https://github.com/fntneves/jquery-labelauty
 * Official website: http://labelauty.js.org/
 * GitHub popularity:
   - Count of stars: 779
   - Count of watchers: 37
   - Count of forks: 100

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by menuka94** [comment]
I'll do the necessary changes. Thanks for replying.

**Comment by menuka94** [comment]
I made the necessary changes. Thanks!

File: ajax/libs/labelauty/1.1.0/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "jquery-labelauty",
+  "version": "1.1.0",
+  "homepage": "https://github.com/fntneves/jquery-labelauty",
+  "authors": [
+    "Francisco Neves <contact@francisconeves.com>"
+  ],
+  "description": "A lightweight and beautiful jQuery plugin for radio and checkbox inputs.",
+  "main": "source/jquery-labelauty.js",
```
Comment (PeterDaveHello) [inline ajax/libs/labelauty/1.1.0/package.json]: Please use `"filename"` instead of `"main"`.

**Comment by menuka94** [comment]
@PeterDaveHello Is there anything else I need to change? Thank you.

File: ajax/libs/labelauty/1.1.0/package.json (hunk 1 above)
Comment (menuka94) [inline ajax/libs/labelauty/1.1.0/package.json]: I made the change. Thanks!

File: ajax/libs/labelauty/1.1.0/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jquery-labelauty",
+  "version": "1.1.0",
+  "homepage": "https://github.com/fntneves/jquery-labelauty",
```
Comment (pvnr0082t) [inline ajax/libs/labelauty/1.1.0/package.json]: We'll use git repo as homepage by default if the custom homepage is not found, so there is no need to set homepage to its git repo.

File: ajax/libs/labelauty/1.1.0/package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  "authors": [
+    "Francisco Neves <contact@francisconeves.com>"
+  ],
```
Comment (pvnr0082t) [inline ajax/libs/labelauty/1.1.0/package.json]: Let's use `"author": "Francisco Neves <contact@francisconeves.com>",` here.

File: ajax/libs/labelauty/1.1.0/package.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  "repository": {
+    "type":"git",
+    "url":"git://github.com/fntneves/jquery-labelauty"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/fntneves/jquery-labelauty"
+  },
```
Comment (pvnr0082t) [inline ajax/libs/labelauty/1.1.0/package.json]: Please ａｌｓｏ add fileMap. You can take a look at https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md.

**Comment by menuka94** [comment]
@pvnr0082t  Thanks for helping me! 
I made the changes. Please let me know if there's anything else. :+1:

File: ajax/libs/labelauty/1.1.0/package.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  "repository": {
+    "type":"git",
+    "url":"git://github.com/fntneves/jquery-labelauty"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/fntneves/jquery-labelauty",
+    "fileMap": [
+      {
+        "basePath": "/source",
```
Comment (pvnr0082t) [inline ajax/libs/labelauty/1.1.0/package.json]: Please remove the leading slash to `"source"`.

File: ajax/libs/labelauty/1.1.0/package.json [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+        "files": [
+         "jquery-labelauty.min.css",
+         "jquery-labelauty.min.js"
```
Comment (pvnr0082t) [inline ajax/libs/labelauty/1.1.0/package.json]: I think we can use the rules as following:
```json
"files": [
  "jquery-labelauty*",
  "images/*.png"
]
```
Thanks.

Comment (menuka94) [inline ajax/libs/labelauty/1.1.0/package.json]: I made the changes. Thanks!

File: ajax/libs/labelauty/1.1.0/package.json [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+        "basePath": "source",
+        "files": [
+         "jquery-labelauty*",
+         "images/*.png"
+        ]
+      }
+    ]
+  },
+  "description": "A lightweight and beautiful jQuery plugin for radio and checkbox inputs.",
+  "filename": "source/jquery-labelauty.js",
```
Comment (PeterDaveHello) [inline ajax/libs/labelauty/1.1.0/package.json]: We should put the files outside `source` directory, as it's basePath, filename should be `"source/jquery-labelauty.js"`

File: ajax/libs/labelauty/1.1.0/package.json [hunk 8]
```diff
@@ -11,10 +11,10 @@
     "target": "git://github.com/fntneves/jquery-labelauty",
     "fileMap": [
       {
-        "basePath": "source",
+        "basePath": "",
```
Comment (PeterDaveHello) [inline ajax/libs/labelauty/1.1.0/package.json]: We should use `source` as basePath, why to remove it?

Comment (menuka94) [inline ajax/libs/labelauty/1.1.0/package.json]: Oh I'm sorry. I thought you asked me to remove it. I'll change it back.

Comment (PeterDaveHello) [inline ajax/libs/labelauty/1.1.0/package.json]: Sorry but I mean to remove it from filename, and put the files outside the `source` directory, we don't need it, thanks.

File: ajax/libs/labelauty/1.1.0/package.json [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/fntneves/jquery-labelauty",
+    "fileMap": [
+      {
+        "basePath": "source",
+        "files": [
+         "source/jquery-labelauty*",
+         "source/images/*.png"
```
Comment (PeterDaveHello) [inline ajax/libs/labelauty/1.1.0/package.json]: Then `source`s here should be removed

File: ajax/libs/labelauty/1.1.0/package.json (hunk 7 above)
Comment (PeterDaveHello) [inline ajax/libs/labelauty/1.1.0/package.json]: @menuka94 sorry that I typed it wrong, I mean filename should not be "source/jquery-labelauty.js"

File: ajax/libs/labelauty/1.1.0/package.json [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/fntneves/jquery-labelauty",
+    "fileMap": [
+      {
+        "basePath": "source",
+        "files": [
+         "jquery-labelauty*",
+         "images/*.png"
```
Comment (PeterDaveHello) [inline ajax/libs/labelauty/1.1.0/package.json]: We could use `images/**/*` here.

**Comment by menuka94** [comment]
@kennynaoh  I have minified the JS and CSS files. Thank you!

**Comment by PeterDaveHello** [comment]
We'll still need the non-minified files here, please add them back with the minified files :)

**Comment by menuka94** [comment]
@PeterDaveHello Done! :)

**Comment by PeterDaveHello** [comment]
`package.json` should be outside the version directory

**Comment by menuka94** [comment]
@PeterDaveHello I moved the file. But now the continuous-integration build fails.

**Comment by PeterDaveHello** [comment]
@menuka94 ci fixed, let's fix the problem it mentions.

File: ajax/libs/labelauty/package.json [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+        "basePath": "source",
+        "files": [
+         "jquery-labelauty*",
+         "images/**/*"
+        ]
+      }
+    ]
+  },
+  "description": "A lightweight and beautiful jQuery plugin for radio and checkbox inputs.",
+  "filename": "jquery-labelauty.js",
```
Comment (pvnr0082t) [inline ajax/libs/labelauty/package.json]: It should point to the main minified file. Use `jquery-labelauty.min.js` here.

File: ajax/libs/labelauty/package.json [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+  "name": "jquery-labelauty",
+  "version": "1.1.0",
+  "author": "Francisco Neves <contact@francisconeves.com>",
+  "repository": {
+    "type":"git",
+    "url":"git://github.com/fntneves/jquery-labelauty"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/fntneves/jquery-labelauty",
```
Comment (pvnr0082t) [inline ajax/libs/labelauty/package.json]: Please use `"git://github.com/fntneves/jquery-labelauty.git"`. Thanks.

File: ajax/libs/labelauty/package.json [hunk 13]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "jquery-labelauty",
+  "version": "1.1.0",
+  "author": "Francisco Neves <contact@francisconeves.com>",
+  "repository": {
+    "type":"git",
+    "url":"git://github.com/fntneves/jquery-labelauty"
```
Comment (pvnr0082t) [inline ajax/libs/labelauty/package.json]: There should be a space after colon mark. https://github.com/cdnjs/cdnjs/blob/5ae759bcf8e719734c6453a5b4feebc7d9832e3d/ajax/libs/labelauty/package.json#L6-#L7

File: ajax/libs/labelauty/package.json [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/fntneves/jquery-labelauty",
+    "fileMap": [
+      {
+        "basePath": "source",
+        "files": [
+         "jquery-labelauty*",
+         "images/**/*"
```
Comment (pvnr0082t) [inline ajax/libs/labelauty/package.json]: https://github.com/cdnjs/cdnjs/blob/5ae759bcf8e719734c6453a5b4feebc7d9832e3d/ajax/libs/labelauty/package.json#L16-L17. Wrong indent.

File: ajax/libs/labelauty/package.json [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+        "basePath": "source",
+        "files": [
+          "jquery-labelauty*",
+          "images/**/*"
+        ]
+      }
+    ]
+  },
+  "description": "A lightweight and beautiful jQuery plugin for radio and checkbox inputs.",
+  "filename": "source/jquery-labelauty.min.js",
```
Comment (pvnr0082t) [inline ajax/libs/labelauty/package.json]: Please keep using `"jquery-labelauty.min.js"` as filename.

Comment (menuka94) [inline ajax/libs/labelauty/package.json]: Ok.

**Comment by menuka94** [comment]
@pvnr0082t ci still fails. Please help me review fixes.

**Comment by menuka94** [comment]
@pvnr0082t could you please help me review the changes?

**Comment by PeterDaveHello** [comment]
oops, should not merge master branch in ...

**Comment by menuka94** [comment]
@PeterDaveHello Sorry. I'm new to open source. But I really want to learn.

**Comment by PeterDaveHello** [comment]
@menuka94 I‘m not fixing this problem, if you'd like to learn, I'll suggest you not to use GitHub gui for PR fixing.

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
Thanks all.

File: ajax/libs/labelauty/package.json [hunk 16]
```diff
@@ -0,0 +1,36 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/fntneves/jquery-labelauty"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/fntneves/jquery-labelauty.git",
+    "fileMap": [
+      {
+        "basePath": "",
```
Comment (PeterDaveHello) [inline ajax/libs/labelauty/package.json]: Still wrong here, should set its basePath here, I'll fix it.

**Comment by menuka94** [comment]
@PeterDaveHello @pvnr0082t @kennynaoh Thank you very much!

**Comment by PeterDaveHello** [comment]
@menuka94 You're welcome, try the next one with git command line :+1:

**Pull request merged**